src/func.d

```diff
@@ -4486,7 +4486,7 @@ public:
         return this;
     }
 
-    override const(char)* kind()
+    override const(char)* kind() const
     {
         // GCC requires the (char*) casts
         return (tok != TOKfunction) ? cast(char*)"delegate" : cast(char*)"function";
```

Comment:
You're in the wrong business for that! BTW, you can also contribute PRs to improve const correctness. I'm only doing it because I am stuck on working on MI because my PRs on it are twisting in the wind: #5402 #5401 #5403